src/detect-engine-address.c

```diff
@@ -456,6 +456,16 @@ static int DetectAddressParseString(DetectAddress *dd, const char *str)
                     goto error;
 
                 netmask = in.s_addr;
+
+                /* validate netmask */
+                int cidr = CIDRFromMask(netmask);
+                if (cidr < 0) {
+                    SCLogError(SC_ERR_INVALID_SIGNATURE,
+                            "netmask \"%s\" is not usable. Only netmasks that are compatible with "
+                            "CIDR notation are supported. See #5168.",
```

Comment:
I'm not sure about this error message.  The ticket has even less information on why this type of netmask is not compatible.  A reference to documentation is likely better.

Reply:
I've updated the ticket to be a bit more clear.